src/pages/index.tsx

```diff
@@ -21,6 +24,18 @@ const Auth = () => {
       <Row justify="center">
         <Astronaut />
       </Row>
+      {isAuth && (
+        <Row justify="center">
+          <Button
```

Comment:
```suggestion
          <Button
```
I think we should put this button in a header